client/src/mergeUtils.js

```diff
@@ -244,6 +246,57 @@ export function unassignedPerson(position1, position2, mergedPosition) {
   }
 }
 
+export function mergedPersonIsValid(mergedPerson) {
+  const msg = []
+  if (!mergedPerson.role) {
+    msg.push("Role")
+  }
+  if (!mergedPerson.status) {
+    msg.push("Status")
+  }
+  if (!mergedPerson.rank) {
+    msg.push(Settings.fields.person.rank)
+  }
+  if (!mergedPerson.gender) {
+    msg.push(Settings.fields.person.gender)
+  }
+  if (!mergedPerson.country) {
+    msg.push(Settings.fields.person.country)
+  }
+  if (mergedPerson.role === Person.ROLE.ADVISOR) {
+    if (!mergedPerson.emailAddress) {
+      msg.push(
+        `${
+          Settings.fields.person.emailAddress.label
+        } for ${Person.humanNameOfRole(Person.ROLE.ADVISOR)} role`
+      )
+    }
+    if (!mergedPerson.endOfTourDate) {
+      msg.push(
+        `${Settings.fields.person.endOfTourDate} for ${Person.humanNameOfRole(
+          Person.ROLE.ADVISOR
+        )} role`
+      )
+    }
+  }
+  if (!_isEmpty(msg)) {
+    const msgContainer = (
+      <div>
+        <div>It is required to fill the following fields</div>
```

Comment:
As his is followed by a list, please add a colon (`:`) at the end: `It is required to fill the following fields:`.